recipes/pourRNA/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+./configure --prefix=$PREFIX
```

Comment:
Can you please do a "${PREFIX}" to be sure, that nothing is messed up with spaces etc